common/transport/amqp/devdoc/amqp_message.md

```diff
@@ -77,20 +77,20 @@ The `toMessage` static method takes an AMQP message from the underlying library
 
-**SRS_NODE_IOTHUB_AMQPMSG_16_004: [** The `toMessage` method shall set the `Message.messageId` property to the `AmqpMessage.properties.messageId` value if it is present. **]**
+**SRS_NODE_IOTHUB_AMQPMSG_16_004: [** The `toMessage` method shall set the `Message.messageId` property to the `AmqpMessage.message_id` value if it is present. **]**
 
-**SRS_NODE_IOTHUB_AMQPMSG_16_005: [** The `toMessage` method shall set the `Message.to` property to the `AmqpMessage.properties.to` value if it is present. **]**
+**SRS_NODE_IOTHUB_AMQPMSG_16_005: [** The `toMessage` method shall set the `Message.to` property to the `AmqpMessage.to` value if it is present. **]**
 
-**SRS_NODE_IOTHUB_AMQPMSG_16_006: [** The `toMessage` method shall set the `Message.expiryTimeUtc` property to the `AmqpMessage.properties.absoluteExpiryTime` value if it is present. **]**
+**SRS_NODE_IOTHUB_AMQPMSG_16_006: [** The `toMessage` method shall set the `Message.expiryTimeUtc` property to the `AmqpMessage.absolute_expiry_time` value if it is present. **]**
 
-**SRS_NODE_IOTHUB_AMQPMSG_16_016: [** The `toMessage` method shall set the `Message.contentType` property to the `AmqpMessage.properties.contentType` value if it is present.  **]**
+**SRS_NODE_IOTHUB_AMQPMSG_16_016: [** The `toMessage` method shall set the `Message.contentType` property to the `AmqpMessage.content_type` value if it is present.  **]**
 
-**SRS_NODE_IOTHUB_AMQPMSG_16_017: [** The `toMessage` method shall set the `Message.contentEncoding` property to the `AmqpMessage.properties.contentEncoding` value if it is present.  **]**
+**SRS_NODE_IOTHUB_AMQPMSG_16_017: [** The `toMessage` method shall set the `Message.contentEncoding` property to the `AmqpMessage.content_encoding` value if it is present.  **]**
 
-**SRS_NODE_IOTHUB_AMQPMSG_16_007: [** The `toMessage` method shall convert the user-defined `AmqpMessage.applicationProperties` to a `Properties` collection stored in `Message.applicationProperties`. **]**
+**SRS_NODE_IOTHUB_AMQPMSG_16_007: [** The `toMessage` method shall convert the user-defined `AmqpMessage.application_properties` to a `Properties` collection stored in `Message.properties`. **]**
 
-**SRS_NODE_IOTHUB_AMQPMSG_16_008: [** The `toMessage` method shall set the `Message.ack` property to the `AmqpMessage.applicationProperties['iothub-ack']` value if it is present. **]**
+**SRS_NODE_IOTHUB_AMQPMSG_16_008: [** The `toMessage` method shall set the `Message.ack` property to the `AmqpMessage.application_properties['iothub-ack']` value if it is present. **]**
 
-**SRS_NODE_IOTHUB_AMQPMSG_16_009: [** The `toMessage` method shall set the `Message.data` of the message to the content of the `AmqpMessage.body` property. **]**
\ No newline at end of file
+**SRS_NODE_IOTHUB_AMQPMSG_16_009: [** The `toMessage` method shall set the `Message.data` of the message to the content of the `AmqpMessage.body.content` property. **]**
```

Comment:
>body.content [](start = 140, length = 12)

is it .body.content or just .content?

Reply:
body.content.

---
In reply to: [191905854](https://github.com/Azure/azure-iot-sdk-node/pull/300#discussion_r191905854) [](ancestors = 191905854)

Reply:
what are the other properties of body then? that's weird

---
In reply to: [191923392](https://github.com/Azure/azure-iot-sdk-node/pull/300#discussion_r191923392) [](ancestors = 191923392,191905854)